01/app.js

```diff
@@ -1,2 +1,10 @@
 const randomNumber = Math.round(Math.random() * 20);
 console.log(randomNumber);
+
+if(randomNumber >=5){
+    for(i=5; i<=randomNumber; i++){
```

Comment:
Zabrakło `let` tj. `for(let i=...` - bez tego tworzymy niepotrzebnie zmienną globalną